source/fundamentals/indexes.txt

```diff
@@ -378,6 +378,21 @@ use the ``createSearchIndex()`` method to create an index called
    :start-after: start createSearchIndex example
    :end-before: end createSearchIndex example
 
+When connecting to {+mdb-server+} v6.0.11, v7.0.2, or later, you can use
```

Comment:
This is funny--it works in specifically 6.0.11, and also 7.0.2 and later? If so, this might make "later" clearer:

```suggestion
When connecting to {+mdb-server+} v6.0.11, or v7.0.2 or later, you can use
```

Reply:
I think it means v6.0.11-latest v6 patch and then v7.0.2-latest v7 patch EXCLUSIVE of 7.0.0 and 7.0.1. I'll work on this phrasing to reflect that.